### DISCUSSION TITLE
Cannot connect to throttles web site.

### DISCUSSION THREAD

**Discussion by Budman1758** [DCC-EX/EX-T3-WiFi#64] [Q&A] [answered]
I cannot seem to connect to the throttles web site no matter how I try. All I get is either "connection refused" or nothing at all. Tried with the throttle connected to my wifi and also with throttle as an AP. 

Also, where is the loco information stored? On the throttle or the command station? I'm using DCC-Ex for the CS.
Pardon my ignorance....Pretty new to DCC.

**Comment by alex-code** [comment, marked as answer]
To access the throttles internal website you need to be on the `T3 WiFi` display (the internal website is only active when on this display).
From the main menu touch the `T3 WiFi` button, the display will update and show the throttles AP details and the details of the WiFi it's connected to (if it's been setup).

Loco information is stored on the MicroSD card that's used in the display.

**Reply by Budman1758** [reply]
OK. Thats better.

**Comment by jrinky** [comment]
I'm having a problem accessing the internal web server.
After accessing the T3 Wifi display I can connect to the throttle, from my laptop but not access the web server it's self, connection refused message.

Laptop firewall turned off.

Ping and port map results are as below. I've mapped specific ports, 80, 8080, 3128, 2560  as well.

[david@localhost ~]$ ping 192.168.4.2
PING 192.168.4.2 (192.168.4.2) 56(84) bytes of data.
64 bytes from 192.168.4.2: icmp_seq=1 ttl=64 time=0.021 ms
64 bytes from 192.168.4.2: icmp_seq=2 ttl=64 time=0.070 ms
^C

[david@localhost ~]$ nmap 192.168.4.2
Starting Nmap 7.93 ( https://nmap.org ) at 2023-02-26 15:56 NZDT
Note: Host seems down. If it is really up, but blocking our ping probes, try -Pn
Nmap done: 1 IP address (0 hosts up) scanned in 3.06 seconds

nmap  192.168.4.2 -p 2560
Starting Nmap 7.93 ( https://nmap.org ) at 2023-02-26 16:04 NZDT
Note: Host seems down. If it is really up, but blocking our ping probes, try -Pn
Nmap done: 1 IP address (0 hosts up) scanned in 3.04 seconds


Any suggestions appreciated.

**Comment by Budman1758** [comment]
> I can connect to the throttle, from my laptop but not access the web server it's self

What exactly do you mean by access the web server itself? If your laptop connects to the throttle then that's the web server serving up its content.

**Reply by jrinky** [reply]
Sorry for not being clearer.
I can establish a network connection to the throttle as per the ping but when using a browser I get the "connection refused" error message. A port scan of the IP shows that there is nothing listening on port 80 which tallies with the connection refused message.

I can confirm it's the correct IP because if I turn the throttle off the ping stops and the network connection drops.

**Reply by jrinky** [reply]
Got it thanks  -- DHCP lease issue. I should have spotted that sooner.
